src/test/java/com/revature/lostchapterbackend/integrationtests/BookIntegrationTest.java

```diff
@@ -0,0 +1,387 @@
+package com.revature.lostchapterbackend;
```

Comment:
package destination is incomplete, it should be:

• package com.revature.lostchapterbackend.integrationtests;

Reply:
fixed it